src/main/resources/music/settings/host/values/strings.xml

```diff
@@ -200,15 +200,15 @@ This does not bypass the age restriction. It just accepts it automatically."</st
     <string name="revanced_ryd_attribution_summary">Data is provided by the Return YouTube Dislike API. Tap here to learn more.</string>
     <string name="revanced_ryd_compact_layout_summary">Hides the separator of the like button.</string>
     <string name="revanced_ryd_compact_layout_title">Compact like button</string>
-    <string name="revanced_ryd_dislike_percentage_summary">Instead of the number of dislikes, the percentage of dislikes is shown.</string>
+    <string name="revanced_ryd_dislike_percentage_summary">Displays the percentage of dislikes instead of the dislike count.</string>
     <string name="revanced_ryd_dislike_percentage_title">Dislikes as percentage</string>
     <string name="revanced_ryd_enabled_summary">Shows the dislike count of videos.</string>
-    <string name="revanced_ryd_failure_connection_timeout">Dislikes temporarily not available (API timed out).</string>
-    <string name="revanced_ryd_failure_connection_status_code" formatted="false">Dislikes not available (status %d).</string>
-    <string name="revanced_ryd_failure_client_rate_limit_requested">Dislikes not available (client API limit reached).</string>
-    <string name="revanced_ryd_failure_generic">Dislikes not available (%s).</string>
-    <string name="revanced_ryd_toast_on_connection_error_summary">Toast is shown if Return YouTube Dislike is not available.</string>
-    <string name="revanced_ryd_toast_on_connection_error_title">Show a toast if API is not available</string>
+    <string name="revanced_ryd_failure_connection_timeout">Dislikes are temporarily unavailable (API timed out).</string>
+    <string name="revanced_ryd_failure_connection_status_code" formatted="false">Dislikes are unavailable (status %d).</string>
+    <string name="revanced_ryd_failure_client_rate_limit_requested">Dislikes are unavailable (client API limit reached).</string>
+    <string name="revanced_ryd_failure_generic">Dislikes are unavailable (%s).</string>
```

Comment:
I'm not sure if adding the word "are" to these toasts makes them too long. I know that toasts will be truncated if they can't fit on-screen which should be considered.